### ISSUE TITLE
ember-cli build system complains about the sourcemaps

### ISSUE THREAD

**Issue by boris-petrov** [airbrake/airbrake-js#309]
We include the non-minified version of airbrake in our builds. Everything was working fine in version 0.5.9. The next release, 0.6.0 broke `ember-cli` (and it's been broken for all versions afterwards). The error is:

```
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, open '.../frontend/tmp/source_map_concat-input_base_path-62UaZV4w.tmp/bower_components/airbrake-js-client/dist/webpack:/webpack/universalModuleDefinition'
```

This is, I guess, since you've included sourcemaps for the non-minified file in the dist. Obviously `ember-cli` doesn't like the `webpack:` thing that is written in it. What should we do?

**Comment by vmihailenco** [comment]
I forgot to reply, but I have no good ideas anyway. Closing since there is no any activity.

**Comment by thec0keman** [comment]
I think there is a legitimate issue here.  It appears the sourcemaps being generated are not valid.
Invalid: [Master](https://sourcemaps.io/validate?url=https%3A%2F%2Fraw.githubusercontent.com%2Fairbrake%2Fairbrake-js%2Fmaster%2Fdist%2Fclient.min.js)
Invalid: [0.6.0](https://sourcemaps.io/validate?url=https%3A%2F%2Fraw.githubusercontent.com%2Fairbrake%2Fairbrake-js%2Fv0.6.0%2Fdist%2Fclient.min.js)
Valid: [0.5.9](https://sourcemaps.io/validate?url=https%3A%2F%2Fraw.githubusercontent.com%2Fairbrake%2Fairbrake-js%2Fv0.5.9%2Fdist%2Fclient.min.js)

I would suggest this issue should be re-opened.

**Comment by boris-petrov** [comment]
@vmihailenco - are you willing to take another look at this issue? You can check [sourcemaps.io](https://sourcemaps.io) - just paste in these three links and see that this is a real issue:

```
https://raw.githubusercontent.com/airbrake/airbrake-js/v0.5.9/dist/client.min.js
https://raw.githubusercontent.com/airbrake/airbrake-js/v0.6.0/dist/client.min.js
https://raw.githubusercontent.com/airbrake/airbrake-js/master/dist/client.min.js
```

The first one is valid according to [sourcemaps.io](https://sourcemaps.io), the second and third aren't.

**Comment by vmihailenco** [comment]
As I understand all files that start with `webpack://` don't exist because:
1. they never existed like `/webpack/universalModuleDefinition` which is the code webpack inserts to load the code, .e.g. https://github.com/airbrake/airbrake-js/blob/master/dist/client.js#L2-L79
2. they were transpiled into JavaScript like `webpack:///./src/reporter/reporter.ts` and don't exist in package bundle

Probably there is a special syntax to indicate that file does not really exist, but I did not google anything yet.

**Comment by boris-petrov** [comment]
Thanks for looking into it!

I don't understand anything of sourcemaps or webpack but there's a ton of resources on the web like [here](https://webpack.js.org/configuration/devtool/), [here](https://webpack.github.io/docs/configuration.html) or [here](https://survivejs.com/webpack/building/source-maps/) that could help. Thanks again for taking the time!

**Comment by vmihailenco** [comment]
Closing as stale.

**Comment by boris-petrov** [comment]
@vmihailenco - this is fixed as of `v2.0.0-beta.2`. Thank you!
